drivers/pn532/pn532.c

```diff
@@ -80,7 +80,7 @@
 
 #if ENABLE_DEBUG
 #define PRINTBUFF printbuff
-static void printbuff(char *buff, unsigned len)
+static void printbuff(uint8_t *buff, unsigned len)
```

Comment:
Is this a leftover of changes related to cppcheck ?

Reply:
I would also revert this and open a separate PR. A least, it should not be part of the same commit containing I2C changes

Reply:
reverted. shall I open the other PR against new_i2c_if too? (master should work though)

Reply:
I think it should be fine against master.